lib/db/schema/patch-026-027.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE IF NOT EXISTS securityEvents (
+  uid BINARY(16) NOT NULL,
+  name VARCHAR(255) NOT NULL,
```

Comment:
The answer whenever the SQL `ENUM` type is mentioned: http://komlenic.com/244/8-reasons-why-mysqls-enum-data-type-is-evil/

If we wanted to go that way, a reference table is better. That just requires an insert to add a new event name. The `ENUM` requires an `ALTER TABLE` (yep...).